### DISCUSSION TITLE
get webhook events on some other apps installation/remove etc

### DISCUSSION THREAD

**Discussion by kfir-dc** [community/community#29669] [Apps, API and Webhooks]
Hey,

we've deployed a GitHub app and I'm trying to get events when some apps are installed/removed/suspended etc from the organization.
seems that we get events only related to our app

https://docs.github.com/en/developers/webhooks-and-events/webhooks/webhook-events-and-payloads#installation

my question is, is there a case that i can get events when app installed/removed on the organization level? 
-rather than webhook/some other event to subscribe to?

we do able to call the API list to retrieve the apps that were installed on the org and their status. but like to be able to get events (and avoid delta checks :) )

if not possible at all, is there some solution lets say i have oAuth app?

thanks

**Comment by ybeezyolifant** [comment]
![IMG_20220710_105549](https://user-images.githubusercontent.com/111915539/186252496-f84b463d-9235-4663-abb8-67b032769052.jpg)
